readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://VitWay.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://VitWay.github.io/layout_html-form/report/html_report/)
```

Comment:
Add these links to the description, please. And actually, they don't work. Check it one more time
<img width="1422" alt="image" src="https://user-images.githubusercontent.com/99403198/216770658-5113fb79-50cf-48ee-a131-45deb4a3518a.png">

Reply:
How to change it? Made according to the instructions

Reply:
Links still not works.
Did you make ```npm run deploy``` ?